src/index/DeltaTriples.cpp

```diff
@@ -150,9 +150,6 @@ void DeltaTriples::modifyTriplesImpl(CancellationHandle cancellationHandle,
                                      TriplesToHandlesMap& targetMap,
                                      TriplesToHandlesMap& inverseMap) {
   rewriteLocalVocabEntriesAndBlankNodes(triples);
-  ql::ranges::sort(triples);
-  auto first = std::unique(triples.begin(), triples.end());
-  triples.erase(first, triples.end());
```

Comment:
If you assume that they are already unique and sorted, then you can AD_EXPENSIVE_CHECK those properties.